### DISCUSSION TITLE
Improve the file naming visibility in File Manager

### DISCUSSION THREAD

**Discussion by Kyndryl-BAndrews** [GitbookIO/community#96] [Feature Requests]
### What's your feature/idea?

Currently if there are duplicate files uploaded to GitBook, they show up as identical names rather than the actual file name. 
Ex. **Image_picture.png** and **Image_picture.png** are both in File Manager, but file names are actually **Image_picture.png** and **Image_picture (1).png**

### What's your desired outcome?

Would like the ability to see the real file names of files in the file manager otherwise it is impossible to know which file is being used in a document and which can be deleted.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
Closing in favour of #234
